gamemodes/terrortown/gamemode/shared/sh_include.lua

```diff
@@ -122,7 +124,7 @@ function ttt_include(filename)
 	local fd = TTTFiles[filename]
 
 	if not fd then
-		error("[TTT2][ERROR] Tried to include a not existant file" .. filename)
+		error("[TTT2][ERROR] Tried to include a not existant file " .. filename)
```

Comment:
I don't think existant exists (no pun intended). non-existent or missing would be better.
Sth. like "[TTT2][ERROR] Tried to include a the non-existent file " .. filename or
"[TTT2][ERROR] Tried to include missing file " .. filename